index.bs

```diff
@@ -204,7 +212,7 @@ with |processor| as parameter.
 
 The <dfn>handleNewFrame</dfn> algorithm is given a |processor| as input.
 It is defined by running the following steps:
-1. If |processor|.`[[maxBufferSize]]` has a value and |processor|.`[[queue]]` has |processor|.`[[maxBufferSize]]` elements, [=queue/dequeue=] an item from |processor|.`[[queue]]`.
+1. If |processor|.`[[queue]]` has |processor|.`[[maxBufferSize]]` elements, [=queue/dequeue=] an item from |processor|.`[[queue]]`.
```

Comment:
You probably need to close the frame that you dequeued

Reply:
If we close it here, we should also use "dequeue and close" on line 219.

Reply:
I'll file an issue about this.
For camera tracks, what we enqueue here is not a WebCodecs VideoFrame object but the necessary information to create one.
For VideoTrackGenerator tracks given to MediaStreamTrackProcessor, we could indeed directly enqueue VideoFrames but we would not expose this VideoFrame object but a clone.